python/pyiceberg/catalog/hive.py

```diff
@@ -159,16 +156,56 @@ def _construct_parameters(metadata_location: str, previous_metadata_location: Op
 def _annotate_namespace(database: HiveDatabase, properties: Properties) -> HiveDatabase:
     params = {}
     for key, value in properties.items():
-        if key == "comment":
+        if key == COMMENT:
             database.description = value
-        elif key == "location":
+        elif key == LOCATION:
             database.locationUri = value
         else:
             params[key] = value
     database.parameters = params
     return database
 
 
+hive_primitive_types = {
```

Comment:
Since this is a constant, should the variable name be all upper case?

Reply:
Good one, copied this one from legacy 👀